acousticbrainz/__init__.py

```diff
@@ -4,7 +4,7 @@
 from logging.handlers import RotatingFileHandler
 
 # This value must be incremented after any schema changes!
-__version__ = 2
+__version__ = 3
```

Comment:
only a schema change on replicated tables. My guess is that we're not going to replicate the user table